bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.html

```diff
@@ -25,13 +24,13 @@
       </div>
     </div>
   </div>
-  <div (click)="$event.stopPropagation();" class="m-1 mr-2">
-    <dt-chart *ngIf="_comparisonView == 'heatmap'"
+  <div (click)="$event.stopPropagation();" class="m-0 mr-2">
+    <dt-chart *ngIf="_showChart && _comparisonView == 'heatmap'"
               [options]="_heatmapOptions"
               [series]="_heatmapSeries"
               (seriesVisibilityChange)="seriesVisibilityChanged($event)">
     </dt-chart>
-    <dt-chart *ngIf="_comparisonView == 'chart'"
+    <dt-chart *ngIf="_showChart && _comparisonView == 'chart'"
```

Comment:
you could wrap the two charts in a ng-container and check for _showChart just onces but for such a simple condition I would consider a matter of personal taste

Reply:
Yep, actually I would do it to with a ng-container, but since I have the *ngIf on each dt-chart anyway, I just thought I would integrate it there.